python/ray/projects/scripts.py

```diff
@@ -177,8 +177,7 @@ def create_cluster(self):
             no_restart=False,
             restart_only=False,
             yes=True,
-            override_cluster_name=self.session_name,
-        )
+            override_cluster_name=self.session_name)
```

Comment:
Ooc, was this intentional?

Reply:
Nope - probably some stray auto-formatting. I'll revert this line for clarity.